crates/re_viewer/src/ui/selection_panel.rs

```diff
@@ -855,7 +855,7 @@ fn blueprint_ui_for_space_view(
             .root_handle()
             .and_then(|root| query_result.tree.lookup_result(root))
         else {
-            re_log::error!("Could not find root data result for Space View {space_view_id:?}");
+            // This can happen when the query was invalid for some reason.
```

Comment:
query doesn't even need to be valid... just empty I think is sufficient to hit this